zookeeper-server/src/test/java/org/apache/zookeeper/test/StandaloneTest.java

```diff
@@ -48,12 +49,18 @@
 
     protected static final Logger LOG = LoggerFactory.getLogger(StandaloneTest.class);
 
-    @Before
+    @BeforeEach
     public void setup() {
         System.setProperty("zookeeper.DigestAuthenticationProvider.superDigest", "super:D/InIHSb7yEEbrWz8b9l71RjZJU="/* password is 'test'*/);
         QuorumPeerConfig.setReconfigEnabled(true);
     }
 
+    @AfterEach
+    @Override
+    public void tearDown() throws Exception{
```

Comment:
please check if we really need this method here

Reply:
Done